packages/core/src/components/overlay/overlay.tsx

```diff
@@ -84,6 +84,11 @@ export interface IBackdropProps {
     backdropProps?: React.HTMLProps<HTMLDivElement>;
 
     /**
+     * Whether underlying content should be scrollable while the backdrop is visible.
+     */
+    backgroundScrollingDisabled?: boolean;
```

Comment:
Why is this configurable? Have you seen (or can you imagine) any use case where `false` would make sense?